src/os/bluestore/BlueStore.cc

```diff
@@ -8314,8 +8314,12 @@ void BlueStore::_dump_extent_map(ExtentMap &em, int log_level)
   }
 }
 
-void BlueStore::_dump_transaction(Transaction *t, int log_level)
+void BlueStore::_dump_transaction(Transaction* const t, const int log_level)
```

Comment:
These const declarations are an overkill IMHO. And they aren't in line with the rest of the bluestore code .